adapters/InMobi/InMobiAdapter/GADMAdapterInMobiInterstitialAd.m

```diff
@@ -123,56 +123,57 @@ - (void)stopBeingDelegate {
 #pragma mark IMAdInterstitialDelegate methods
 
 - (void)interstitialDidFinishLoading:(nonnull IMInterstitial *)interstitial {
-    NSLog(@"<<<< interstitialDidFinishRequest >>>>");
+    GADMAdapterInMobiLog(@"InMobi SDK loaded a interstitial ad successfully.");
     _interstitalAdEventDelegate = _interstitialRenderCompletionHandler(self, nil);
 }
 
 - (void)interstitial:(nonnull IMInterstitial *)interstitial
 didFailToLoadWithError:(IMRequestStatus *)error {
+    GADMAdapterInMobiLog(@"InMobi SDK fail to load interstitial ad.");
     _interstitialRenderCompletionHandler(nil, error);
 }
 
 - (void)interstitialWillPresent:(nonnull IMInterstitial *)interstitial {
-    NSLog(@"<<<< interstitialWillPresentScreen >>>>");
+    GADMAdapterInMobiLog(@"InMobi SDK will present a full screen modal view from a interstitial ad.");
```

Comment:
fix the the log here in int ad and rewarded ad
InMobi SDK will present a full screen interstitial ad

Reply:
same for all parallel callnbacks in int and rewarded ads